src/runtime/vendor/go.opentelemetry.io/otel/verify_examples.sh

```diff
@@ -36,7 +36,7 @@ if ! git diff --quiet; then \
 	exit -1
 fi
 
-if [ "$(git tag --contains $(git log -1 --pretty=format:"%H"))" = "" ] ; then
```

Comment:
I don't think we should update vendor code

Reply:
ah... my bad. I will update the PR.